### DISCUSSION TITLE
`NavigationView.IsBackEnabled` is not showing the back arrow in Android, works find in windows, ios ect

### DISCUSSION THREAD

**Discussion by cconner100** [unoplatform/uno#15570] [Q&A] [answered]
### Current behavior

Made a simple program to set the NavigationView as follows
`<NavigationView x:Name="NavView"
                   PaneDisplayMode="LeftCompact"
                   IsBackButtonVisible="Auto"
                   IsBackEnabled="True">
<NavigationView.MenuItems>
  <NavigationViewItem Content="Home" Icon="Home"/>
  <NavigationViewItem Content="Profile" Icon="Contact"/>
  <NavigationViewItem Content="Settings" Icon="Setting"/>
</NavigationView.MenuItems>
<Frame x:Name="ContentFrame" />
</NavigationView>`

This shows the back button on Windows, iOS and Mac fine but it does not seem to show on Android.  Searched online and could not find a question about this.

Attached is a simple repro

### Expected behavior

The navigationview should show the back arrow on android
[TestAndriodNavView.zip](https://github.com/unoplatform/uno/files/14357913/TestAndriodNavView.zip)

**Comment by MartinZikmund** [comment, marked as answer]
You can use the [`IsBackButtonVisible` property](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.navigationviewbackbuttonvisible?view=windows-app-sdk-1.4) to adjust this. By default it is set to `Auto`, which should follow the target system guidelines. In the case of Android, the app should not display a back button in the UI, as it is not per the [design guidelines](https://developer.android.com/guide/navigation/navigation-custom-back).

> All Android devices provide a Back button for this type of navigation, so don't add a Back button to your app’s UI.

So by default, the back button is not shown. You can override this by setting `IsBackButtonVisible` to `Visible` instead

**Reply by MartinZikmund** [reply]
I have added documentation to cover this here https://github.com/unoplatform/uno/pull/15571

**Comment by cconner100** [comment]
OK understand and it works but not sure why it does not follow the same rules as the other controls.

The logic is now, for Non android set the IsBackButtonVisible to auto.  then set the BackEnabled based on the frame.cangoback.

But for Android I just set the IsBackButtonVisible to frame.IsBackEnabled. seems kind of strange to have this workaround to do the same thing.
